spec/models/item_spec.rb

```diff
@@ -37,7 +37,7 @@
     expect(item).to be_invalid
   end
 
-  xit 'returns an empty item array' do
+  it 'returns an empty categories array' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.